src/style.css

```diff
@@ -1,3 +1,86 @@
+/* stylelint-disable rule-empty-line-before */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --color-blue: #00acdc;
+  --color-black: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.link {
+  text-decoration: none;
+  color: var(--color-black);
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  border-radius: 8px;
+  height: 4px;
+  width: 100%;
+  z-index: 9;
+  background: var(--color-blue);
+}
+
+.link.is-active,
+.link:hover {
+  color: var(--color-blue);
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 30px 0 50px;
```

Comment:
The padding values are not consistent. You should make sure the horizontal padding is consistent on both sides.